Data/Models/Class.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace mPole.Data.Models
+{
+    public class Class
+    {
+        [Key]
+        public int Id { get; set; }
+        public required string Duration { get; set; }
+        public required DateTime Date { get; set; }
+        public required string Location { get; set; }
+        public required string Trainer { get; set; }
+        public int TrainingId { get; set; }
+        public Training Training { get; set; } = null!;
+        public IList<string> RegisteredUsers { get; set; } = new List<string>(); //w postaci lsity username'ów starczy? nie chce całych userów, żeby uniknąć joinowej tabeli
```

Comment:
Hmm jeżeli to jest tabela bazodanowa to lepiej powiązać One To Many z tabelą Userów wg mnie mimo wszystko, Twoje rozwiązanie być może zadziała, ale jest mało modularne w tabeli w bazie danych, możesz rozważyć też trzymanie tego w pamięci